app/vmselect/promql/exec.go

```diff
@@ -62,6 +70,14 @@ func Exec(qt *querytracer.Tracer, ec *EvalConfig, q string, isFirstPointOnly boo
 	if err != nil {
 		return nil, err
 	}
+	if !isSubQueryComplete(e, false) {
+		if *disableImplicitConversion {
+			return nil, fmt.Errorf("query contains incomplete subquery, which is forbidden when `-search.disableImplicitConversion=true`")
```

Comment:
we should link the doc here, so user could read what to do with the query

Reply:
done.